config/jobs/kubernetes/system-validators/OWNERS

```diff
@@ -8,12 +8,14 @@ reviewers:
 - rosti
 - ereslibre
 - yastij
+- pacoxu
```

Comment:
same as above (but for reviewer)

Reply:
@pacoxu please only keep fabrizio, you and me for both the reviewer and approver list.